client/storage.c

```diff
@@ -491,7 +491,7 @@ void xdag_init_storage(xdag_time_t start_time, xdag_time_t end_time, void *data,
 /* Calls a callback for all blocks from the repository that are in specified time interval; returns the number of blocks */
 uint64_t xdag_load_blocks(xdag_time_t start_time, xdag_time_t end_time, void *data, void *(*callback)(void *, void *))
 {
-	struct xdag_block buf[bufsize], *pbuf[bufsize];
+	struct xdag_block *buf, *pbuf[bufsize];
```

Comment:
Did you meet no memory issue with this section?

Reply:
Stack size depend system, on some system xdag crash for this.

Reply:
Could you point out which system? I would like know the specific details before confirm code changes.

Reply:
OpenBSD 6.3